main.html

```diff
@@ -0,0 +1,362 @@
+              class="text"
+              alt="shoe-text"
+            />
+          </div>
+        </div>
+      </div>
+
+      <div id="prev" class="fas fa-chevron-left"></div>
+      <div id="next" class="fas fa-chevron-right"></div>
+    </section>
+
+    <section class="service">
+      <div class="box-container">
+        <div class="box">
+          <i class="fas fa-shipping-fast"></i>
+          <h3>Fast Delivery</h3>
+          <p>
+            Get your products delivered in superfast mode. Under 90 Mins at
+            unbelievable prices from the best sellers
+          </p>
+        </div>
+
+        <div class="box">
+          <i class="fas fa-undo"></i>
+          <h3>10 days replacements</h3>
+          <p>
+            Replacement is now easy with us. 10 days. Refund, Replacement or
+            Exchange.
+          </p>
+        </div>
+
+        <div class="box">
+          <i class="fas fa-headset"></i>
+          <h3>24 x 7 support</h3>
+          <p>
+            Available throughout the day, 6 days a week.This simply means that
+            the technical service is available around the clock.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="latest-products" id="products">
+      <h1 class="heading">Latest Collections</h1>
+
+      <div class="products-container">
+        <div class="product-box">
+          <img src="./components/images/feature-1.jpg" alt="product-photo" />
+          <div class="content-product">
+            <h3 class="desc-products">
+              Get your Jackets now at Best Price. For Limited Period Only.
+            </h3>
+            <a href="#" class="btn">Explore More</a>
+          </div>
+        </div>
+
+        <div class="product-box">
+          <img src="./components/images/feature-3.jpg" alt="product-photo" />
+          <div class="content-product">
+            <h3 class="desc-products">
+              Get your Travel Bags now at Best Price. Offer only for Today!
+            </h3>
+            <a href="#" class="btn">Explore More</a>
+          </div>
+        </div>
+
+        <div class="product-box">
+          <img src="./components/images/feature-2.jpg" alt="product-photo" />
+          <div class="content-product">
+            <h3 class="desc-products">
+              Get your Hiking Bags now at Best Price. Grab it now.
+            </h3>
+            <a href="#" class="btn">Explore More</a>
+          </div>
+        </div>
+
+        <div class="product-box">
+          <img src="./components/images/feature-4.jpg" alt="product-photo" />
+          <div class="content-product">
+            <h3 class="desc-products">
+              Get your Hiking Boots now at Best Price. For Limited Period Only.
+            </h3>
+            <a href="#" class="btn">Explore More</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="latest-brands" id="brands">
+      <h1 class="heading">Featured Brands</h1>
```

Comment:
There should be only one `h1` element per HMTL page. Read about [heading here](https://www.contentkingapp.com/academy/headings/)